### DISCUSSION TITLE
EOF while parsing a value when watching a release mode

### DISCUSSION THREAD

**Discussion by saito-kosuke** [cargo-lambda/cargo-lambda#371] [Q&A]
When I execute `cargo lambda watch --release`, the error below occurred after processing.

ERROR Lambda runtime invoke{requestId="b59bac63-6570-4f70-97d1-4563c0db7935"}: Error("EOF while parsing a value", line: 1, column: 0)

When I execute `cargo lambda watch`, the error didn't occur.
Could someone tell me how to deal with that?

**Comment by davehorner** [comment]
@saito-kosuke created this discussion and this ticket https://github.com/cargo-lambda/cargo-lambda/issues/372; just linking issue on this discussion.  Both discussion and issue remain.
